wasm.md

```diff
@@ -254,6 +254,14 @@ Structured Control Flow
     rule <k> nop => . ... </k>
 ```
 
+`unreachable` causes an immediate `trap`.
+
+```k
+    syntax Instr ::= "unreachable"
+ // -----------------------
```

Comment:
Please line the `---` underneath up with the end of the line above.

Sorry, I'm a stickler for formatting though :P

Reply:
Nothing to apologize for, I appreciate well though-out styling.